compiler/onnx-subgraph/Readme.md

```diff
@@ -0,0 +1,48 @@
+## onnx_autosubgraphs
```

Comment:
It seems we have similar tool already with circle model, not ONNX.
But I think you can refer compiler/circle-partitioner/README.md file for explaining what and how to use the tool.